### ISSUE TITLE
Beds: formspec isn't updated when some player leaves game

### ISSUE THREAD

**Issue by ssdaniel24** [minetest/minetest_game#3126]
**How to reproduce:**

1. There are 4 players on a server at night: A, B, C and D.
2. Two of them (A, B) lie down on the beds and sleep. They see "*2 of 4 players are sleeping*".
3. If third player (C) will disconnect from the server, sleeping players (A, B) will see same sign - "*2 of 4 players are sleeping*" and will not see "skip night" button.

Extra: But if fourth player (D) will lie down on the bed too, then formspec will be updated ("*3 of 3 players are sleeping*") and night will be skipped with button or automatically.

**Expected behavior:**

3. If third player (C) will disconnect from the server, sleeping players (A, B) will see updated sign - "*2 of 3 players are sleeping*" and will able to skip night automatically or using button.

**Comment by alexx2021** [comment]
Hello, I would like to try and solve this issue. Can I please be assigned to it?

**Comment by appgurueu** [comment]
> Hello, I would like to try and solve this issue. Can I please be assigned to it?

I don't think we assign issues here. Just open a PR when you've solved it. It suffices that you've expressed interest in working on it here.
